### PR TITLE
Feature/welcome screen

### PR DESCRIPTION
**Pull request by belengg27**

### CODE REVIEW

**Review by Hamzezi — APPROVED** [review]
Looks good to me

File: app/build.gradle [hunk 1]
```diff
@@ -153,6 +153,9 @@ dependencies {
     androidTestImplementation 'androidx.test:runner:1.4.0'
     androidTestImplementation 'junit:junit:4.13.2'
 
+
+
+
```
Comment (yanisdb) [inline app/build.gradle:158]: A bit wierd the empty lines

File: app/src/androidTest/java/ch/proximeety/proximeety/presentation/MainActivityTest.kt [hunk 2]
```diff
@@ -22,8 +24,18 @@ class MainActivityTest {
 
     @Test
     fun sampleTest() {
```
Comment (yanisdb) [inline app/src/androidTest/java/ch/proximeety/proximeety/presentation/MainActivityTest.kt:26]: Should rename the test

**Review by yanisdb — APPROVED** [review]
Very nice. I added a few minor comments but I approve the pull request.

**Pull request merged**